espanol/pensamiento/guerra.md

```diff
@@ -8,3 +8,7 @@ Tras terminar con sus lamentaciones:
 [Una reflexi�n de gran importancia le asalt�](estoicismo/estoicismo.md)
 
 [Se sentó en el suelo y se puso a meditar](meditacion/meditar.md)
+
+Cuando volvi� a mirar al cielo, se dio cuenta de que estaba cubierto de nubes. �Con tan malos auspicios comenzar�a la batalla?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.